cinema/views.py

```diff
@@ -1,16 +1,15 @@
 from datetime import datetime
 
 from django.db.models import F, Count
+from drf_spectacular.utils import extend_schema, OpenApiParameter
```

Comment:
The import of `extend_schema` and `OpenApiParameter` from `drf_spectacular.utils` is correct and allows for detailed API documentation. Ensure that all endpoints are documented for better API usability.